ipfs-cluster/test_tests/README.txt

```diff
@@ -0,0 +1,9 @@
+These tests are used for sanity checking the DSL implementation of node
```

Comment:
You may want to move these files up one level, as it's not ipfs-cluster specific.